app/views/RoomView/index.js

```diff
@@ -67,18 +67,28 @@ export default class RoomView extends LoggedView {
 		const t = navigation.getParam('t');
 		const tmid = navigation.getParam('tmid');
 		const isFetching = navigation.getParam('isFetching', false);
+		const unreadsCount = navigation.getParam('unreadsCount') > 0 ? navigation.getParam('unreadsCount') : null;
 		return {
-			headerTitleContainerStyle: styles.headerTitleContainerStyle,
-			headerTitle: (
-				<RoomHeaderView
-					rid={rid}
-					prid={prid}
-					tmid={tmid}
-					title={title}
-					type={t}
-					widthOffset={tmid ? 95 : 130}
-					isFetching={isFetching}
-				/>
+			headerLeft: (
```

Comment:
Placing the entire header on the left button feels wrong and may be impacted by future react-navigation updates.